### PR TITLE
WebUI Services, fix edit, delete button on top of each other, generate more compact table

### PR DESCRIPTION
**Pull request by fbourqui**
Fix edit, delete button on top of each other in Services page, after update to 22.2.0
generate a more compact table removing big empty space after last service of each device.

#### Please note

> Please read this information carefully. You can run `./lnms dev:check` to check your code before submitting.

- [x] Have you followed our [code guidelines?](https://docs.librenms.org/Developing/Code-Guidelines/)
- [x] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.
- [ ] If my Pull Request makes discovery/polling/yaml changes, I have added/updated [test data](https://docs.librenms.org/Developing/os/Test-Units/).

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

Screenshot
Before:
Services-before

After:
Services-After

### CODE REVIEW

File: includes/html/pages/services.inc.php [hunk 1]
```diff
@@ -169,7 +169,6 @@
                         if ($service_iteration < 2 && $header) {
                             echo '<div class="panel panel-default">';
                             echo '<div class="panel-heading"><h3 class="panel-title">' . $devlink . '</h3>' . $device_sysName . '</div>';
-                            echo '<div class="panel-body">';
```
[the remark this answers is not in the document]
Comment (fbourqui) [reply includes/html/pages/services.inc.php:172]: Hello,

I've though of that, but looking at other tables there is always one table/list per page, not several, ex device list:
Devices-only-one-table-list-pe-page

If I re-add the panel body, there is that ugly gap at the end of each table, hupper space of the table is a little larger too:
 
Services-Keeping-pannel-body

Compare to the nice looking proposition:
Services-After

File: includes/html/pages/services.inc.php [hunk 2]
```diff
@@ -236,13 +235,8 @@
                         echo '</tr>';
 
                         if ($service_iteration >= $services_count) {
-                            $footer = true;
```
Comment (murrant) [inline includes/html/pages/services.inc.php:239]: You left the $footer initialization up above.

File: includes/html/pages/services.inc.php (hunk 1 above)
[the remark this answers is not in the document]
Comment (murrant) [reply includes/html/pages/services.inc.php:172]: I prefer the extra spacing on the top and sides, but yes, the bottom spacing looks a little silly.  All are fine to me.

**Review by fbourqui — COMMENTED** [review]
@PipoCanaja 
There is another panel-body that enclose the all list host/services, host/services, ...
At line 104                 echo '<div class="panel-body">';
( I couldn't figure out how to make my comment be lined to the line.

And if we look at Devices page there is only one level of panel-body.

File: includes/html/pages/services.inc.php (hunk 1 above)
[the remark this answers is not in the document]
Comment (fbourqui) [reply includes/html/pages/services.inc.php:172]: @PipoCanaja 
There is another panel-body that enclose the all list host/services, host/services, ...
At line `104                 echo '<div class="panel-body">';`
( I couldn't figure out how to make my comment be linked to the line.)

And if we look at Devices page there is only one level of panel-body.

File: includes/html/pages/services.inc.php (hunk 2 above)
Comment (fbourqui) [reply includes/html/pages/services.inc.php:239]: OK, $footer initialization removed

File: includes/html/pages/services.inc.php (hunk 1 above)
[the remark this answers is not in the document]
Comment (fbourqui) [reply includes/html/pages/services.inc.php:172]: Guys, 
One more question while we are reviewing the service page look/design, would you like to remove the "Remote Host" Column.
As all view (All, OK, Warning, Critical) group services by "Remote host", so the column doesn't make sense to me.
What do you think should we remove it ?

[the remark this answers is not in the document]
Comment (fbourqui) [reply includes/html/pages/services.inc.php:172]: Added commit to remove the "Remote Host" column.

**Comment by PipoCanaja** [comment]
Hi @fbourqui 
Remote host does not have to be the same for all services under a device. I have a "GoogleDNS" device I created, under which I have 2 DNS services, one against ipv4 and one against ipv6. So RemoteHost is different, device is the same. 
Capture d’écran 2022-02-27 à 23 53 52

**Comment by fbourqui** [comment]
Hi @PipoCanaja, 
I've reverted the change, re-added Remote host. but I resized the minimum width of Name (you seem to not use it like me), and also Remote host.

**Comment by PipoCanaja** [comment]
> Hi @PipoCanaja, I've reverted the change, re-added Remote host. but I resized the minimum width of Name (you seem to not use it like me), and also Remote host.

It is not because you don't use it, and I have it empty in the capture I sent you, that nobody else is using it (me included :) )  So please stick with what was supposed to be fixed by this PR in the 1st place : 
- button alignment
- more compact table

When done, please attach a new screenshot. 

Thanx

**Comment by fbourqui** [comment]
I understand, that I'm not the only one using librenms, and I overlooked the effect of changing the minimum size on low width display.

I will make a separate pull request for increasing the max width of Message, Description, and decrease the size of Last Changed, when/if #13817 is merged.

I revered all column size change.

image

**Comment by fbourqui** [comment]
@PipoCanaja, @murrant Could you review, thanks.

**Comment by murrant** [comment]
#14071 supersedes this.

**Pull request closed without merging**